secboot/secboot_tpm.go

```diff
@@ -43,27 +44,38 @@ import (
 
 const (
 	// Handles are in the block reserved for owner objects (0x01800000 - 0x01bfffff)
-	pinHandle = 0x01880000
+	pinHandle           = 0x01880000
+	policyCounterHandle = 0x01880001
+
+	// keyctl(2) key identifiers
+	sessionKeyring = -3
+	userKeyring    = -4
```

Comment:
I see `KEY_SPEC_SESSION_KEYRING` and `KEY_SPEC_USER_KEYRING` in https://pkg.go.dev/golang.org/x/sys/unix

Reply:
Thanks, using these instead.